src/lib/executions.js

```diff
@@ -30,28 +30,19 @@ function ExecutionsClient( config ){
  *
  * @description
  * returns a list of executions
```

Comment:
improve docs by using {object} and then specifying fields -
```
@param {object} opts
@param {string} opts._include
@param {string} opts.deployment_id
```

for now - just the relevant fields for us. 
we will think how to document all the available options later.. not important for now. 

this documentation style is important for stuff such as auto-complete etc..